generators/client/templates/angularjs/src/test/javascript/spec/app/account/settings/_settings.controller.spec.js

```diff
@@ -35,7 +35,8 @@ describe('Controller Tests', function() {
                 activated: true,
                 email: "john.doe@mail.com",
                 langKey: "<%= nativeLanguage %>",
-                login: "john"
+                login: "john",
```

Comment:
this needs to be done only for angular not for angular1 hence please remove changes in the `angularjs` folder